miden/src/examples/mod.rs

```diff
@@ -50,11 +51,18 @@ pub struct ExampleOptions {
 //#[clap(about = "available examples")]
 pub enum ExampleType {
     /// Compute a Fibonacci sequence of the specified length
-    Fib {
+    Fibonacci {
```

Comment:
nit: I'd probably keep the old name (unless there is a good reason to change it?).

Reply:
I decided to change it because this name defines what name we will use in the CLI call. So instead of 
`miden example fib -n 1000`
we could call
`miden example fibonacci -n 1000`
I thought that name `fib` could be unclear, but it is not a big deal — I can change it back to the old name.

Reply:
I think `fib` is fine as it is easier to type :)